CHANGES.md

```diff
@@ -1,5 +1,10 @@
 ## Changes in 0.10.2 (in development)
 
+* Added support for Landsat-4,5 Level-2 (`"LTML2"`) 
+  and Landsat 7 Level-2 (`"LETML2"`) collections
+* Adapted to work with Python 3.10
+* Changed default time tolerance from ten months to ten minutes
```

Comment:
This is a suboptimal phrasing. The intention was never 10 months (you should know that).

```suggestion
* Fixed default `time_tolerance` used when `time_period` is None. It is now correctly set to ten minutes.
```

Reply:
If it really was 10 months, then we should have almost always received entirely wrong results. But that was obviously not the case. Strange.